lib/organisms/gravity-form/gravity-form.provider.js

```diff
@@ -1,66 +1,66 @@
-(function () {
+angular
+    .module('lnPatterns')
+    .provider('$lnOGravityFormConfig', lnOGravityFormConfigProvider);
 
-    /**
-     * Created by Chris on 13/04/2016.
-     */
+/**
+ *
+ * Expects: {
+ *      api_base: 'http://www.urltoapi.com/api',
+ *      api_key: 'public-api-key',
+ *      forms: {
+ *          '1': {
+ *              'get_form': {
+ *                  'route': 'forms/1',
+ *                  'signature: 'serverGeneratedSignatureString',
+ *                  'expires: 'serverGeneratedExpiresTimestamp'
+ *              },
+ *              'post_submission: {
+ *                  'route': 'forms/1/submissions'
+ *              }
+ *          },
+ *          '2': {
+ *              ...
+ *          },
+ *          ...
+ *      }
+ *  }
+ */
 
-    'use strict';
-
-    /**
-     *
-     * Expects: {
-     *      url: 'http://www.urltoapi.com/api',
-     *      key: 'public-api-key',
-     *
-     *      TODO: decide if this:
-     *        sigUrl: 'http://www.apitoretrievesignatureurl.com/'
-     *      OR this:
-     *        sigCallback: pointerToProvidedMethod
-     *      is appropriate
-     *  }
-     */
-
-    var REQUIRED_KEYS = [
-        'url',
-        'key',
-        'sigUrl',
-        'sigCallback'
-    ];
-
-    function lnOGravityFormConfigProvider() {
-        var
-            self = this,
-            _config = null
-        ;
+/*@ngInject*/
+function lnOGravityFormConfigProvider() {
+    var
+        REQUIRED_KEYS = [
+            'api_base',
+            'api_key',
+            'forms'
+        ],
+        self = this,
+        _config = null
+    ;
 
-        this.setConfig = function (config) {
-            _config = config;
-            _config.isValid = self.hasValidConfig();
-        };
+    this.setConfig = function (config) {
+        _config = config;
+        _config.isValid = self.hasValidConfig();
+    };
 
-        this.hasValidConfig = function () {
-            var _hasParams = true;
-            for (var i = 0; i < REQUIRED_KEYS.length; i++) {
-                var prop = _config[REQUIRED_KEYS[i]];
+    this.hasValidConfig = function () {
+        var _hasParams = true;
+        for (var i = 0; i < REQUIRED_KEYS.length; i++) {
+            var prop = _config[REQUIRED_KEYS[i]];
 
-                if (typeof prop === "undefined") {
-                    _hasParams = false;
-                    break;
-                }
+            if (typeof prop === "undefined") {
```

Comment:
I think he trying to say that this double quotes should be changed to single quotes.